test/test_findCardMatch.py

```diff
@@ -0,0 +1,11 @@
+from testing_base import *
+
+def test_find_card_match():
+    deck1 = Deck()
+    deck1.shuffle()
+    player1 = Player("john")
+    player2 = Player("jeff")
+    player1CardList = []
+    player2CardList = []
+
+    findCardMatch()
```

Comment:
Is this missing an assert function for pytest?

Reply:
What exactly is the assert function?